tests/integration/spm/test_build.py

```diff
@@ -61,5 +61,43 @@ def test_spm_build(self):
         # Make sure formula path dir is created
         self.assertTrue(os.path.isdir(self.config['formula_path']))
 
+    def test_spm_build_big_file(self):
+        '''
+        test spm build
+        '''
+        big_file = self.run_function('cmd.run',
+                                     ['fallocate -l 1G {0}'.format(os.path.join(self.formula_sls_dir,
```

Comment:
Should we gate this test around the existence of the `fallocate` binary?

Reply:
done. great idea :)